specification/2.0/Specification.adoc

```diff
@@ -539,17 +696,23 @@ The following example defines a buffer. The `byteLength` property specifies the
 }
 ----
 
-A *bufferView* represents a subset of data in a buffer, defined by a byte offset into the buffer specified in the `byteOffset` property and a total byte length specified by the `byteLength` property of the buffer view.
+The byte length of the referenced resource **MUST** be greater than or equal to the `buffer.byteLength` property.
+
+Buffer data **MAY** alternatively be embedded in the glTF file via `data:` URI with base64 encoding. When `data:` URI is used for buffer storage, its `mediatype` field **MUST** be set to `application/octet-stream` or `application/gltf-buffer`. The byte length of the base64-encoded data **MUST** be equal to the `buffer.byteLength` property.
```

Comment:
Requiring an external file containing the binary buffer to have an exact size seems too restrictive as applications may want to store something (e.g. a checksum) at the end of it. This use case does not affect glTF operations. I don't think it's practical to do same with `data:`-URIs, but we should probably allow it anyway.

Thoughts?

Reply:
I don't know exactly whether there may be a case that somebody wants to add, for example, some padding bytes to the data for whatever reason. The main point of the question was 1. I didn't see a reason for the difference between `>=` for buffer and `==` for data URI, and 2. I didn't see a strong reason to not also allow `>=` for the latter. But I don't have strong opinions here. 

(In doubt, one can argue to start with the "more strict" version, because _relaxing_ a requirement later is usually far easier than _strengthening_ a requirement - and unless there's a use case for something like "padded data URI data", we can leave it as it is)

Reply:
Removed the equality requirement since it prevents buffer storage type conversion.